quarto/about.qmd

```diff
@@ -2,4 +2,31 @@
 title: About the Project
 ---
 
-Page content coming soon!
+The falcon project is a product of industry collaborations to encourage the development of shared code, expertise, knowledge and products.
+
+In the near future, the goal is to develop functionality to create regulatory reporting tables suitable for FDA filing. The long-term objective of this collaboration is to develop an R package that can be used industry wide - developing a cross-industry standard and reducing manual work from a reporting perspective.
+
+## Our Collaboration Journey
+
+Current collaborators are from Roche, Sanofi, Boehringer-Ingelheim, and Moderna. Developers are connected via GitHub for the creation of shared solutions using publicly available analysis packages for R. Developers are invited to browse the falcon project repository and related packages for insights into our template designs.
+
+### Joining as a Collaborator
+
+We invite further collaboration with additional companies joining this initiative and contributing to the falcon project. Please feel free to contact any of the repository maintainers for additional information and check out the section below for more information on joining the project as a developer.
+
+#### Information for Developers
+
+As a pharmaverse collaboration, we invite any interested developers to join the falcon team and contribute to template development. To on-board as a developer, please reach out to a product owner (Vincent Shen, Freeman Wang, Lian Lin or Thomas Neitman) to receive access to the falcon GitHub repository.
```

Comment:
Maybe also link the email address of the PO's?
Should we also mention Juergen Boehl as PO and is Thomas still involved? I thought he left Roche :)

And we could also link to the GitHub repo again.